src/rolls.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+function Rolls() {
```

Comment:
Use domain language to name your objects.  We don't talk about _rolls_ in bowling, we talk about frames and games.